storageapi/src/vespa/storageapi/messageapi/returncode.cpp

```diff
@@ -145,5 +167,21 @@ ReturnCode::isBucketDisappearance() const
             return false;
     }
 }
+namespace {
+    vespalib::stringref Empty("");
+}
+vespalib::stringref
+ReturnCode::getMessage() const {
+    return _message ? _message->operator vespalib::stringref() : Empty;
```

Comment:
Is the global object really needed here, vs. just returning a default-constructed `stringref`?

Reply:
No, you are right. Old habit....